pom.xml

```diff
@@ -25,7 +25,7 @@
     <useBeta>true</useBeta>
 
     <!-- Jenkins Plug-in Dependencies Versions -->
-    <checks-api.version>0.2.2</checks-api.version>
+    <checks-api.version>0.2.3-rc36.2f1d4eb96d17</checks-api.version>
```

Comment:
Release a 0.2.3 to include the `toString` method feature, I'm not sure if that a single PR worth a release?

Reply:
I wouldn’t worry about what deserves a release, just release whenever you’re ready.